src/icalendar/tests/test_issue_168_parsing_invalid_calendars_no_warning.py

```diff
@@ -0,0 +1,8 @@
+'''Issue #168 - Parsing invalid icalendars fails without any warning
+
+   https://github.com/collective/icalendar/issues/168
+'''
+from icalendar import Calendar
+
+def test_issue_168_parsing_inavlid_calendars_no_warning(calendars):
+    assert calendars.issue_168_input.to_ical() == calendars.issue_168_expected_output.raw_ics
```

Comment:
Maybe an additional test to ensure that vevents-errors is not empty but contains the error for the wrong line. (Maybe this is tested in a different place.)

Reply:
This is now included. Thanks!